### DISCUSSION TITLE
How can I show more detailed logs at `tail -f -n 50 otelcol-output.log`

### DISCUSSION THREAD

**Discussion by AllanOricil** [open-telemetry/opentelemetry-collector#9906] [General]
I have `cloudwatch` receiver configured but I dont see any log messages regarding it working

<img width="379" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector/assets/55927613/48dc7a54-59f7-42a5-b8a1-1e4ee7acd3bb">

<img width="1484" alt="image" src="https://github.com/open-telemetry/opentelemetry-collector/assets/55927613/e27ccdfe-1e27-4c51-988f-c85f4ddb7361">

Im using otel-collector `otelcol-contrib version 0.88.0`

**Comment by atoulme** [comment]
See https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md
